tasks/lib/FileName.js

```diff
@@ -42,3 +42,56 @@ module.exports.toString = function(file) {
   fileName = gutil.replaceExtension(path.basename(fileName), '');
   return fileName.replace(/_/g, ' ');
 };
+
+/**
+* Return the next file in alphabetical orded
+*/
+module.exports.nextFile = function(file) {
+  let fileName;
+  let nextFileName;
+  if (typeof file === 'string') {
+    fileName = file;
+  } else if (typeof file.path === 'string') {
+    fileName = path.basename(file.path);
+  } else {
+    return '';
+  }
+  fileName = gutil.replaceExtension(path.basename(fileName), '');
+
+  var ampFiles = getAmpFilesFromDir('src')
+  ampFiles.sort(function(file1, file2){
+   var fileName1=path.basename(file1.path).toLowerCase(), 
+    fileName2=path.basename(file2.path).toLowerCase();
+   //sort string ascending
+   if (fileName1 < fileName2)
+    return -1;
+   if (fileName1 > fileName2)
+    return 1;
+   return 0;
+  });
+
+  for (var i = 0; i < ampFiles.length -1 ; i++) {
+    var currentFileNameWithoutExtension = gutil.replaceExtension(path.basename(ampFiles[i]), '')
+    if (currentFileNameWithoutExtension == fileName) {
+      nextFileName = path.basename(ampFiles[++i]);
+    }
+  }
+  return nextFileName;
+};
+
+function getAmpFilesFromDir(dir) {
+  var filesystem = require("fs");
+  var fileResults = [];
+  filesystem.readdirSync(dir).forEach(function(file) {
+    file = dir+'/'+file;
+    var stat = filesystem.statSync(file);
+
+    if (stat && stat.isDirectory()) {
+      //avoid other folders like img and template
+      if(path.basename(file).startsWith('amp')){
+        fileResults = fileResults.concat(getAmpFilesFromDir(file))
+      }
+    } else fileResults.push(file);
+  });
+  return fileResults;
+}
```

Comment:
I think it is ok to ignore directories for now. Let that be my problem when I add the sections (and we know how they are structured. This is what an idiomatic findNext version in node could look like:

```
function nextFile(dir, currentFile) {
  const files = fs.readdirSync(dir).sort().filter(function(file) { 
    return path.extname(file) == '.html'; 
  });
  const pos = files.indexOf(currentFile);
  if (pos == -1) {
    return null;
  }
  return files[pos+1];
}
```